tests/backup/backup_upgrade_test.go

```diff
@@ -248,7 +248,7 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", func() {
 		restoreNames                       []string
 		mutex                              sync.Mutex
 	)
-
+	var backupToContextMapping = make(map[string][]*scheduler.Context, 0)
```

Comment:
Can we put this along with other variables?

Reply:
done